index.html

```diff
@@ -12,39 +12,52 @@
+                        <li><a href="mailto: contact.popey17@gmail.com"><i class="fa-regular fa-envelope"></i></a></li>
+                    </ul>
+                </div>
+                <div><a href="#"><i class="fa-solid fa-bars"></i></a></div>
+            </div>
         </nav>
     </header>
-    <div class="nav-items">
-        <ul>
-            <li><a href="#">Portfolio</a></li>
-            <li><a href="#">About</a></li>
-            <li><a href="#">Contact</a></li>
-            <li><a href="#"><i class="fa-regular fa-envelope"></i></a></li>
-        </ul>
-    </div>
     <!--Home Section-->
-    <section class="home">
-        <div class="intro">
-            <h1>Hey There. I'm Leo</h1>
-        </div>
-        <div class="job">
-            <h2>I'am a full-stack Web developer</h2>
-        </div>
-        <div class="bio">
-            <p>I can help your business improve by building a nice website or portfolio. You can check some of my recent
-                works and if you have something to ask ,don't hestiate to contact me. Hope we can work together.</p>
-        </div>
-        <div class="socials">
-            <ul>
-                <li><a href=""><i class="fa-brands fa-github"></i></a></li>
-                <li><a href=""><i class="fa-brands fa-linkedin-in"></i></a></li>
-                <li><a href=""><i class="fa-brands fa-angellist"></i></a></li>
-                <li><a href=""><i class="fa-brands fa-twitter"></i></a></li>
-                <li><a href=""><i class="fa-brands fa-medium"></i></a></li>
-            </ul>
+    <section>
+        <div class="home-section">
+            <div class="home">
+                <div class="home-content">
+                    <div class="intro">
+                        <h1>Hey There. I'm Leo</h1>
+                    </div>
+                    <div class="job">
+                        <h2>I'am a full-stack Web developer</h2>
+                    </div>
+                    <div class="bio">
+                        <p>I can help your business improve by building a nice website or portfolio. You can check some
+                            of
+                            my recent
+                            works and if you have something to ask ,don't hestiate to contact me. Hope we can work
+                            together.
+                        </p>
+                    </div>
+                </div>
+                <div class="socials">
+                    <ul>
+                        <li><a href=""><i class="fa-brands fa-github"></i></a></li>
+                        <li><a href=""><i class="fa-brands fa-linkedin-in"></i></a></li>
+                        <li><a href=""><i class="fa-brands fa-angellist"></i></a></li>
+                        <li><a href=""><i class="fa-brands fa-twitter"></i></a></li>
+                        <li><a href=""><i class="fa-brands fa-medium"></i></a></li>
+                    </ul>
+                </div>
+            </div>
```

Comment:
[OPTIONAL] You have done a great in making your project responsive😉 Kindly take note of the intro section😉 you can add a bit of bottom padding to make your project similar to the design😉 Kindly take note of the image below😎

Your project
<img width="893" alt="your desktop" src="https://user-images.githubusercontent.com/64643995/220838020-ef587f46-139a-4424-960e-9662d907383a.png">

design
<img width="509" alt="design_desktop" src="https://user-images.githubusercontent.com/64643995/220838032-8e841731-6d0e-4e3c-af94-9a605be36977.png">